web/src/app/components/Modal/EditStudentProfile.tsx

```diff
@@ -0,0 +1,97 @@
+import { Tabs, Box, TextInput, Textarea, Button, Select, Divider } from '@mantine/core';
+import { AboutTab } from '../Tabs/AboutTab';
+import { EducationTab } from '../Tabs/EducationTab';
+import { SkillsTab } from '../Tabs/SkillsTab';
+import { CVTab } from '../Tabs/CVTab';
+import styles from '../../styles/Modal.module.css';
+import { useMediaQuery } from '@mantine/hooks';
+import {useState} from 'react'
+
+export const EditStudentProfile = () => {
+  const [activeTab, setActiveTab] = useState('about');
+
+  const tabOptions = [
+    { value: 'about', label: 'About Me' },
+    { value: 'education', label: 'Education' },
+    { value: 'skills', label: 'Skills' },
+    { value: 'cv', label: 'CV' },
+  ];
+
+  const isMobile = useMediaQuery('(max-width: 430px)'); //mobile screen 
+
+  const renderContent = () => {
```

Comment:
Smart use of the Tab components, please ensure that the tabs are not too far apart from each other.

Reply:
I have changed the distance. Please let me know if you want them to be closer